demos/serverless/deploy.js

```diff
@@ -41,6 +45,10 @@ function usage() {
   console.log(`  -a, --application                    Browser application to deploy, default '${app}'`);
   console.log(`  -e, --event-bridge                   Enable EventBridge integration, default is no integration`);
   console.log(`  -c, --chime-endpoint                 AWS SDK Chime endpoint, default is '${chimeEndpoint}'`);
+  console.log(`  -iad, --chime-iad-endpoint           AWS SDK Chime IAD endpoint, default is '${chimeMeetingsIADEndpoint}'`);
```

Comment:
I'm not sure I understand the design here. In order to make this extensible, you should have single argument `-rp/--regional-endpoint <value>`

Reply:
We need all of them as we need to switch between them based upon url parameter `controlRegion`.